### PR TITLE
Simplify PR template for ease of use

### PR DESCRIPTION
**Pull request by hmacdonald-verta**
The Folks™️ wanted a simpler, less wordy PR template.

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -1,18 +1,9 @@
-<!-- Title of the PR must comply with [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) guidelines. --> 
-<!-- Title should include the JIRA ticket in square brackets after the conventional commit prefix. This will automatically link the PR in JIRA. -->
-<!-- Example: "fix: [JIRA-123] Allow creation of groups with no members" -->
-
-## Impact and Context
-<!-- Brief description of why these changes are necessary. Don't rewrite the entire Jira ticket. -->
-
-## Risks and Area of Effect
```
Comment (eheinlein-verta) [inline pull_request_template.md:8]: I think the Impact and Context and Risks and Area of Effect headings could stay, just with the comments removed to make it cleaner.

File: pull_request_template.md [hunk 2]
```diff
@@ -1,18 +1,9 @@
...
 ## Testing
-<!-- Explain how this contribution has been tested, e.g. what tests were added and where they have been run. -->
+- [ ] Unit test
+- [ ] Deployed to dev env
+- [ ] Other (explain) 
 
-## How to Revert
-<!-- List steps required to revert this change. For example, note if we'd need to revert liquibase changes. -->
+## Reverting
+- [ ] Contains Migration - _Do Not Revert_
```
Comment (eheinlein-verta) [inline pull_request_template.md:9]: 😍

**Review by eheinlein-verta — APPROVED** [review]
I made a suggestion but it's just that, not something I feel super strongly about.

**Review by jkwatson-verta — APPROVED** [review]
Thanks!

**Pull request merged**